addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -276,7 +276,21 @@ const Wysiwyg = Widget.extend({
         this.$editable.on('click', '.o_image, .media_iframe_video', e => e.preventDefault());
         this.showTooltip = true;
         this.$editable.on('dblclick', mediaSelector, function () {
-            if (this.isContentEditable || (this.parentElement && this.parentElement.isContentEditable)) {
+            let isEditable =
+                // TODO that first check is probably useless/wrong: checking if
+                // the media itself has editable content should not be relevant.
+                // In fact the content of all media should be marked as non
+                // editable anyway.
+                this.isContentEditable ||
```

Comment:
I think you're right. This is probably an artifact from a previous implementation. I'd remove it while you're at it.

Reply:
Alright 👍 Changing `isMediaElement` was part of the task's original spec. I think I remember it was part of the diff at some point. It seems to have gotten lost in one of the reworkings back then.